index.js

```diff
@@ -3,64 +3,23 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 var restify = require('restify')
-var error = require('../error')
-var bufferize = require('../bufferize')
-var config = require('../config')
-var log = require('../log')(config.logLevel, 'db-api')
-var DB = require('../db/mysql')(log, error)
-var pkg = require('../package.json')
+var bufferize = require('./bufferize')
+var version = require('./package.json').version
 
-var MemoryMonitor = require('../memory_monitor')
-var memoryMonitor = new MemoryMonitor()
-memoryMonitor.on(
-  'mem',
-  function (usage) {
-    log.stat(
-      {
-        stat: 'mem',
-        rss: usage.rss,
-        heapTotal: usage.heapTotal,
-        heapUsed: usage.heapUsed
-      }
-    )
-  }
-)
-memoryMonitor.start()
-
-process.on(
-  'uncaughtException',
-  function (err) {
-    log.fatal({
-      op: 'uncaughtException',
-      err: err
-    })
-    process.exit(8)
-  }
-)
-
-function shutdown() {
-  process.nextTick(process.exit)
-}
-
-// defer to allow ass code coverage results to complete processing
-if (process.env.ASS_CODE_COVERAGE) {
-  process.on('SIGINT', shutdown)
-}
-
-function startServer(db) {
+function createServer(db) {
 
   function reply(fn) {
     return function (req, res, next) {
       fn.call(db, req.params.id, req.body)
         .then(
           function (result) {
-            log.info(
+            api.emit(
+              'success',
               {
-                op: 'request.summary',
-                code: 200,
                 route: req.route.name,
                 method: req.method,
-                path: req.url,
+                url: req.url,
+                result: result,
```

Comment:
Might this `result` var contain sensitive information, e.g. tokens etc?  I'm not yet sure where the things emitted from this object will end up, but we should be careful not to leak sensitive data.